WcaOnRails/app/controllers/api/v0/api_controller.rb

```diff
@@ -112,6 +112,37 @@ def delegates
     paginate json: User.delegates
   end
 
+  def records
+    concise_results_date = Timestamp.find_by(name: "compute_auxiliary_data_end").date
+    cache_key = "records/#{concise_results_date.iso8601}"
+    json = Rails.cache.fetch(cache_key) do
```

Comment:
Cool! Is this our first time using `Rails.cache` like this? What kind of cache is this? Is this stored in memory, or on the filesystem somewhere?

Totally unrelated to this PR, but would this be appropriate to use instead of the memoization pattern we use translations_controller? https://github.com/jfly/worldcubeassociation.org/blob/6bd4258e99237907cc30410acfd35af7610679b6/WcaOnRails/app/controllers/translations_controller.rb#L6-L13

Reply:
`Rails.cache` is a place where you can store anything you like, just have a look at [this](http://guides.rubyonrails.org/caching_with_rails.html#low-level-caching).

Concerning the way we do it for translations, I think we should absolutely use the same technique. Taking class attribute as the cache (as we do now) has two downsides I can think of:

1. It gets lost whenever the app is restarted (e.g. on every deploy), whereas a significant number of deploys doesn't involve translation changes.
2. It's being stored in the memory and we'd rather avoid having more memory consumed.

The only thing we need is some cache key that would invalidate whenever a translation changes (digest of a file or something like this). We could even make it more atomic, so that changing a single locale wouldn't invalidate the whole cache, so the computation would take longer only when the English locale changes.

Reply:
I think it would be fine to use something like `WcaOnRails::BOOTED_AT` in the cache key, so it gets invalidated every time we restart the server. I don't think we need to do anything more clever to avoid recomputing translation state when restarting the server!

Reply:
Right, makes sense, at least we would free our memory from holding all this stuff =)